snapcraft/internal/sources.py

```diff
@@ -310,6 +346,10 @@ def __init__(self, source, source_dir, source_tag=None,
     def provision(self, dst, clean_target=True, keep_zip=False):
         zip = os.path.join(self.source_dir, os.path.basename(self.source))
```

Comment:
@sergiusens This is applicable to the original source as well and should be dealt with in a separate PR, I'm just using what has already been given to me for variable names.